src/renderer/lib/migrations.js

```diff
@@ -95,15 +91,15 @@ function migrate_0_7_0 (saved) {
 }
 
 function migrate_0_7_2 (saved) {
-  if (!saved.prefs) {
```

Comment:
Yeah, don't want to get in a habit of changing these after-the-fact, but it looked weird that every migration had a different way of checking for undefined (`!condition` vs. `=== undefined` vs. `== null`).

What we really want is `== null`, IMO.

Now that it's consistent, it's clear what to use in future migrations.